lib/rake/proxmox.rb

```diff
@@ -224,9 +224,17 @@ def define
 
           desc 'list backup jobs'
           task 'cluster:backup:list' do
-            puts 'list_backup_jobs: '
-            proxmox.list_backup_jobs.each do |c|
-              puts " content:#{c}"
+            puts 'list of current backup jobs: '
+            proxmox.fetch_backup_jobs.each_with_index do |c, i|
+              puts "#{i + 1}.  id: #{c['id']}; starttime: #{c['starttime']}"
+            end
+          end
+
+          desc 'list parameters for backup job'
+          task 'cluster:backupjob:listparameter', %i[jobid] do |_t, args|
```

Comment:
all backupjob related tasks should be within same rake task namespace. What do you think about putting them all in `cluster:backupjob`? 
Please move `cluster:backup:list` to `cluster:backupjob:list`.